db/sg_replicate_cfg.go

```diff
@@ -252,6 +264,28 @@ func (rc *ReplicationConfig) Equals(compareToCfg *ReplicationConfig) (bool, erro
 	return bytes.Equal(currentBytes, compareToBytes), nil
 }
 
+// Redact returns the ReplicationCfg with auth credentials of the remote database
+// redacted. Credentials will be redacted from replication config and remote URL.
+func (r *ReplicationCfg) Redact() (config *ReplicationCfg) {
+	config = r
+	if r.Username != "" {
+		config.Username = ""
+	}
+	if r.Password != "" {
+		config.Password = ""
```

Comment:
Is it worth knowing that there was a username/password specified, instead of emptying the value? Something like replacing the values with `"****"`